src/main/java/seedu/duke/template/LrTemplate.java

```diff
@@ -92,6 +92,7 @@ public String toString() {
      */
     @Override
     public LoadComponent getComponent(String component) throws DukeException {
+        assert component.equals("r") || component.equals("l");
```

Comment:
Not sure if tested through JUnit, but if build gradle fails checks again after removing the other statement, then this is likely causing the issue too.

Reply:
Update: Consider removing this

Reply:
Sure, will remove it.